WALLET.md

```diff
@@ -117,140 +145,55 @@ bundle install
 bundle exec pod install
 ```
 
-If your machine does not recognize the `gem` command, you may need to [download Ruby](https://rubyinstaller.org/) first.
-
 1. Run `yarn install` in the repository root.
 2. Run `yarn dev:ios` in the repository root.
 
-And the app should be running in the simulator! If you run into any issues, see below for troubleshooting.
-
-#### Install on an M1
-
-Currently it is not possible to install the wallet natively. There are a few problems that need to be addressed before being able to run the repo on an M1:
-
-1. The M1 comes with a preinstalled version of Ruby that doesn't work with this repository.
-
-2. The build process that gets executed with `yarn dev:ios` is not able to finish with `nvm` installed.
-
-3. Running the necessary scripts with an M1-native node version will not work.
-
-4. It is currently not possible to run the repository in the integrated VSCode terminal.
+#### Install Rosetta
 
-5. It is not possible to run the repository when the containing folder is located anywhere within `Documents` or `Desktop`.
-
-##### Solution
-
-1. Make sure all necessary software (VS Code, Terminal/iTerm2, XCode, Simulator) is running with Rosetta. You can verify this by typing `arch` in your terminal.
-
-```console
-$ arch
--> i386
-```
-
-2. Reinstall `brew` with x86_64 architecture:
-
-```console
-arch -x86_64 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
-```
-
-3. (optional) add your different brew installation paths as aliases to your `.zshrc`:
-
-```bash
-alias ibrew='arch -x86_64 /usr/local/bin/brew'
-alias mbrew='arch -arm64e /opt/homebrew/bin/brew'
-```
-
-4. Install [`rbenv`](https://github.com/rbenv/rbenv) on Intel architecture:
-
-```console
-ibrew install rbenv
-```
-
-4.1. Add rbenv initialization to your shell. For instance in `.zshrc` or `.bashrc`:
+If you are unable to run the app in the iOS Simulator, install Rosetta:
```

Comment:
after installing Rosetta, did you need to do anything? e.g. restart xcode or terminal using Rosetta? i think i'm no longer using Rosetta so i can't remember this detail